cmd/configure/localization/de.toml

```diff
@@ -136,3 +136,13 @@ ChargeModeNone = "Keinen (Benutzt den im Ladepunkt gesetzten Lademodus)"
 Site_Setup = "- Standort einrichten"
 Site_Title = "Titel des Standortes"
 Site_DefaultTitle = "Mein Zuhause"
+
+Site_MaxCurrent = "Wie ist der Hausanschluss abgesichert (Haussicherung in Ampere)?"
+Site_MaxCurrent35A = "35A (max 22kW)"
+Site_MaxCurrent50A = "50A (max 31kW)"
+Site_MaxCurrent63A = "63A (max 39kW)"
+Site_MaxCurrent80A = "80A (max 50kW)"
+Site_MaxCurrent100A = "100A (max 62kW)"
+Site_MaxCurrentCustom = "anderer Wert, manuelle Eingabe"
+Site_MaxCurrent0A = "unbekannt (kein Lastmanagement)"
```

Comment:
unknown oder inaktiv?

Reply:
Bedeutet "kein Lastmanagement". Es wird dann "0" in die config geschrieben.
Kann das "unbekannt" wegnehmen